overcloud-node-growvols.yml

```diff
@@ -0,0 +1,82 @@
+---
```

Comment:
This file seems to have code similar to https://github.com/openstack/tripleo-ansible/blob/master/tripleo_ansible/playbooks/cli-overcloud-node-growvols.yaml with parameter values changed. In order to make it easier to maintain in the future, I think it would be better to have a task which replaces the required values in `/usr/share/ansible/tripleo-playbooks/cli-overcloud-node-growvols.yaml` directly instead of storing this file in Jetpack and copying it later.

Reply:
Yes.
Actually, that's also not the correct way of doing it. I was going through the upstream docs and we can make use of extra_vars to achieve this. I'll amend the commit in a while, testing this with that approach.